app/views/location.scala.html

```diff
@@ -55,5 +57,7 @@
         )
 
         @govukButton(Button(content = Text(messages("site.continue"))))
+
+        @goodsLocationExpander()
```

Comment:
I've looked at the ticket and in the screenshots the expander is before 'Continue' button.

Reply:
Ah yes, good spot :+1: